src/extract-major-beta-version.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+
+import os, re, sys
+
+from github import Github, InputGitAuthor, enable_console_debug_logging
+
+
+#
+# All these are lifted from relbot/utils.py
+#
+
+
+def major_version_from_release_branch_name(branch_name):
+    if matches := re.match(r"^releases[_/]v(\d+)\.0\.0$", branch_name):
```

Comment:
We're using this regex in a number of places. Using the `releases/` format is no longer supported; we could update these to just support `releases_`.

Reply:
Sure! Thank you!